From 0517503b21875a7ebd5c87d139dcf7957e915bb2 Mon Sep 17 00:00:00 2001
From: Tom Marshall <tdm.code@gmail.com>
Date: Thu, 17 Jan 2019 15:16:59 -0800
Subject: [PATCH 4/5] Revert "resize: Make a target for standalone
 resize2fs_static binary"

This reverts commit 2688e2d90febed87f13b982e58933aa7bafee412.

This breaks e2fsdroid, all-sudbir-makefiles only looks one level deep.

The simple fix would be to add an Android.mk to contrib, but we will be
adding resize2fs to recovery fstools using Android.bp.  So this is not
needed.

Change-Id: I40ed659e657be5b2d5544ac7a247ad2b92b98e62
---
 Android.mk        |  1 -
 resize/Android.mk | 35 -----------------------------------
 2 files changed, 36 deletions(-)
 delete mode 100644 Android.mk
 delete mode 100644 resize/Android.mk

diff --git a/Android.mk b/Android.mk
deleted file mode 100644
index 5053e7d6..00000000
--- a/Android.mk
+++ /dev/null
@@ -1 +0,0 @@
-include $(call all-subdir-makefiles)
diff --git a/resize/Android.mk b/resize/Android.mk
deleted file mode 100644
index fa288182..00000000
--- a/resize/Android.mk
+++ /dev/null
@@ -1,35 +0,0 @@
-LOCAL_PATH := $(call my-dir)
-
-resize2fs_src_files := \
-	extent.c \
-	resize2fs.c \
-	main.c \
-	online.c \
-	sim_progress.c \
-	resource_track.c
-
-resize2fs_c_includes := external/e2fsprogs/lib
-
-resize2fs_cflags := -O2 -g -W -Wall
-
-resize2fs_shared_libraries := \
-	libext2fs \
-	libext2_com_err \
-	libext2_e2p \
-	libext2_uuid \
-	libext2_blkid
-
-resize2fs_system_shared_libraries := libc
-
-include $(CLEAR_VARS)
-
-LOCAL_SRC_FILES := $(resize2fs_src_files)
-LOCAL_C_INCLUDES := $(resize2fs_c_includes)
-LOCAL_CFLAGS := $(resize2fs_cflags)
-LOCAL_STATIC_LIBRARIES := $(resize2fs_shared_libraries)
-LOCAL_STATIC_LIBRARIES += $(resize2fs_system_shared_libraries)
-LOCAL_MODULE := resize2fs_static
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_PATH := $(PRODUCT_OUT)/install/bin
-LOCAL_FORCE_STATIC_EXECUTABLE := true
-include $(BUILD_EXECUTABLE)
-- 
2.17.1

